### PR TITLE
feat: Add indicator and indicator type models

### PR DESCRIPTION
**Pull request by jimiolaniyan**
This PR adds support for querying Indicators and Indicator types.

### CODE REVIEW

File: src/main/java/org/hisp/dhis/model/Indicator.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import lombok.Setter;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@Getter
+@Setter
+public class Indicator extends NameableObject
+{
+    @JsonProperty
+    IndicatorType indicatorType;
```
Comment (larshelge) [inline src/main/java/org/hisp/dhis/model/Indicator.java:13]: Add `private` modifier.

File: src/main/java/org/hisp/dhis/model/IndicatorType.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import lombok.Setter;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@Getter
+@Setter
+public class IndicatorType extends NameableObject
+{
+    @JsonProperty
+    Integer factor;
```
Comment (larshelge) [inline src/main/java/org/hisp/dhis/model/IndicatorType.java:13]: Add private modifier.

File: src/main/java/org/hisp/dhis/model/IndicatorType.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+    @JsonProperty
+    Boolean number;
```
Comment (larshelge) [inline src/main/java/org/hisp/dhis/model/IndicatorType.java:16]: Add private modifier.

**Review by larshelge — APPROVED** [review]

File: src/main/java/org/hisp/dhis/model/IndicatorType.java (hunk 3 above)
Comment (jimiolaniyan) [reply src/main/java/org/hisp/dhis/model/IndicatorType.java]: All added now @larshelge.

**Pull request merged**